src/sql/workbench/contrib/executionPlan/browser/executionPlanPropertiesViewBase.ts

```diff
@@ -99,13 +111,27 @@ export abstract class ExecutionPlanPropertiesViewBase implements IVerticalSashLa
 		this._headerContainer = DOM.$('.header');
 		propertiesContent.appendChild(this._headerContainer);
 
+		this._searchAndActionBarContainer = DOM.$('.search-action-bar-container');
+		propertiesContent.appendChild(this._searchAndActionBarContainer);
+
 		this._headerActionsContainer = DOM.$('.table-action-bar');
-		propertiesContent.appendChild(this._headerActionsContainer);
+		this._searchAndActionBarContainer.appendChild(this._headerActionsContainer);
 		this._headerActions = new ActionBar(this._headerActionsContainer, {
 			orientation: ActionsOrientation.HORIZONTAL, context: this
 		});
 		this._headerActions.pushAction([new SortPropertiesByDisplayOrderAction(), new SortPropertiesAlphabeticallyAction(), new SortPropertiesReverseAlphabeticallyAction()], { icon: true, label: false });
 
+		this._propertiesSearchInputContainer = DOM.$('.table-search');
+		this._propertiesSearchInputContainer.classList.add('codicon', searchIconClassNames);
+		this._propertiesSearchInput = new InputBox(this._propertiesSearchInputContainer, this._contextViewService, {
```

Comment:
Register this (and make this class disposable - then make sure it itself is registered by anything that creates it and so on...)

Make sure you go through this class and register any other disposables you create too (like the onDidChange handler below)